assets/css/custom.css

```diff
@@ -350,29 +360,155 @@ input[type="text"].letters-input {
 }
 
 
-/* Bag Switcher */
+/* bag style */
 
-.bags {
+.bag-style {
   display: flex;
   justify-content: center;
   align-items: center;
   flex-direction: row;
   height: 100%;
 }
 
-.bag-thumbs-cnr {
+.bag-style-thumbs-cnr {
   display: block;
   Height: 20%;
 }
 
-.bag-thumb {
+.bag-style-thumb {
   height: 50%;
 }
 
-.bag-thumb img {
+.bag-style-thumb img {
   height: 90%;
 }
 
+.bag-style-cnr {
+  height: 40vw;
+  width: 100%;
+  display: flex;
+  margin: auto;
+  align-items: center;
+  justify-content: flex-start;
+  flex-direction: row;
+}
+
+.bag-style-main {
+  background: url('../img/bags/danny-black.png') no-repeat center;
+  background-size: contain;
+  width: 70%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+}
+
+.bag-style-details-thumb-cnr {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  width: 15%;
+}
+
+.bag-style-details-thumb {
+  height: 25%;
+}
+
+.bag-style-details-thumb img {
+  height: 90%;
+}
+
+
+/*
```

Comment:
try not to commit commented out code. always try to follow the YAGNI principle http://wiki.c2.com/?YouArentGonnaNeedIt